mirror-godot-app/ui/common/components/endless_scroll_flow_container.gd

```diff
@@ -69,6 +69,8 @@ func _calculate_max_items_on_screen(forced_size = Vector2.ZERO) -> int:
 
 
 func _fetch_data(cnt_items_to_fetch: int) -> Array:
+	if not fetch_callable.is_valid():
+		return []
 	assert(fetch_callable.is_valid())
```

Comment:
@RevoluPowered Please investigate this before merge, the assert was being hit in the new engine builds.

<img width="840" alt="Screenshot 2024-06-02 at 3 20 16 AM" src="https://github.com/the-mirror-gdp/the-mirror/assets/1646875/e4fd74ca-824a-42bc-acc6-3c5ed5a7e2f3">

Reply:
Good to know checking this out now.

Reply:
it was a race condition caused by the code running before the user logged in. pushing up fix

Reply:
Fixed in my commit :) was simple